src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -97,6 +97,12 @@ public override void CleanUp()
 
         protected override Task CreateVersionsAsync(CancellationToken cancellationToken)
         {
+            // allVersions is null if server doesn't return any versions.
```

Comment:
Fix the comment.  Here it mentions `allVersions`.  I think you moved the existing comment below up here.  Here it should be `_allPackageVersions`.  Or you can avoid the problem altogether with:
```C#
// The value will be null if the server does not return any versions.
```